src/SIM/SIMbase.C

```diff
@@ -1343,10 +1344,21 @@ bool SIMbase::solutionNorms (const TimeDomain& time,
 	for (k = 0; k < ssol.size(); k++)
           if (ssol[k].empty())
             norm->getProjection(k).clear();
-          else
-            this->extractPatchSolution(ssol[k],norm->getProjection(k),lp-1,nCmp,1);
+          else {
+            if (this->fieldProjections()) {
```

Comment:
Better with `else if (this->fieldProjections()) {`

Reply:
the lack of bracing here makes the code utterly unreadable. combine that with tabs all over the place and i was happy when i managed to get something that compiled. changed.